src/content/docs/release-notes/agent-release-notes/ruby-release-notes/ruby-agent-720.mdx

```diff
@@ -3,6 +3,9 @@ subject: Ruby agent
 releaseDate: '2021-07-02'
 version: 7.2.0
 downloadLink: 'https://rubygems.org/downloads/newrelic_rpm-7.2.0.gem'
+features: ["Configuration of expected/ignored errors by class name, status code, and message now avaliable"]
+bugs: ["Add additional logic to avoid conflicts with use of Module#Prepend", "kwarg support fixed for Rack middleware instrumentation", "Resolve error when adding a new key into hash during iteration"]
```

Comment:
```suggestion
bugs: ["Add additional logic to avoid conflicts with use of Module#Prepend", "Fix kwarg support for Rack middleware instrumentation", "Resolve error when adding a new key into hash during iteration"]
```